doc/technotes/errorHandling.rst

```diff
@@ -0,0 +1,190 @@
+.. _readme-errorHandling:
+
+==============
+Error Handling
+==============
+
+This README describes the error handling feature as currently implemented.
```

Comment:
Let's make some sort of reference to CHIP  or something to CHIP 8 (you could just say "CHIP 8 describes the complete design and rationale").